soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
If you want to change the behavior I'm going to do another look over to try and find more places it could be used. I would rather we kept the old behavior though, I'll try and see why it needed to be changed.